scss/6-components/_navigation.scss

```diff
@@ -1,7 +1,52 @@
 @use 'sass:map';
-@import 'navigation_greedy-nav';
 
-/** @define navigation */
+// ============================================================================
```

Comment:
I've moved all the navigation styles including single-use mixins into this file as they were previously split accross three.